packages/nextjs/app/builders/0x4b2b0D5eE2857fF41B40e3820cDfAc8A9cA60d9f/page.tsx

```diff
@@ -0,0 +1,103 @@
+"use client";
```

Comment:
You can remove the use client here, since you are not using any client capabilities... so the page will be completely pre-rendered in the server.

Reply:
Oh, I see... I'll delete that line